scripts/update-i18n.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+set -e
+
+PWD=$(dirname "${BASH_SOURCE[0]}")
+cd $PWD/..
+
+if [ ! -d .tx ]; then
+    echo "please make sure this script is under the scripts directory!"
+    exit -9
+fi
+
+if [ "$1" != "-v" ]; then
+    QUIET_FLAGS="-silent"
+fi
+
+function regenerate_source() {
+    SEAFILE_PROJECT="seafile-client.pro"
+    rm -f $SEAFILE_PROJECT
+    qmake -project -o $SEAFILE_PROJECT
+    lrelease $SEAFILE_PROJECT $QUIET_FLAGS
+    lupdate $SEAFILE_PROJECT $QUIET_FLAGS
+    rm -f $SEAFILE_PROJECT
+}
```

Comment:
It seems to me [this change](https://github.com/haiwen/seafile-client/commit/f18789950425a852e905b3cb531f880d87ef914b#diff-af3b638bc2a3e6c650974192a53c7291L340) has something to do with #372 . 

"update-ts" doesn't work properly in current master, but with the above change reverted, "update-ts" works without any problem.